test/doppelganger-test.js

```diff
@@ -19,40 +20,33 @@ describe("Doppelganger", function () {
   it("cannot mint the snapshot if you are not the owner", async function () {
     [nftOwner, notNftOwner] = await ethers.getSigners();
 
-    // Mint the original NFT
-    const txn = await mockContract.connect(nftOwner).mint("http://google.com/nfts");
-    await txn.wait();
-
-    // Get the original NFT tokenId
-    let tokensCount = await mockContract.balanceOf(nftOwner.address)
-    const tokenId = await mockContract.tokenOfOwnerByIndex(nftOwner.address, tokensCount - 1);
+    // prime mock - owner addr of this token
+    const tokenId = 1;
+    myFake.ownerOf.returns(nftOwner.address);
 
     // Minting the snapshot should fail if you're not the owner
     await expect(
-      doppelgangerContract.connect(notNftOwner).snapshot(mockContract.address, tokenId)
+      doppelgangerContract.connect(notNftOwner).snapshot(myFake.address, tokenId)
     ).to.be.revertedWith("Doppelganger: only owner of original token can snapshot.");
   });
 
   it("can mint the snapshot if you are the owner", async function () {
     [nftOwner] = await ethers.getSigners();
 
-    // Mint the original NFT
-    const txn = await mockContract.connect(nftOwner).mint("http://google.com/nfts");
-    await txn.wait();
-
-    // Get the original NFT tokenId
-    let tokensCount = await mockContract.balanceOf(nftOwner.address)
-    let tokenId = await mockContract.tokenOfOwnerByIndex(nftOwner.address, tokensCount - 1);
+    // prime mock - owner addr of this token
+    const tokenId = 1;
+    myFake.ownerOf.returns(nftOwner.address);
```

Comment:
Could maybe put this in setup too since it's shared across the tests?

Reply:
change is here: 
https://github.com/johnmosesman/nftrusty/pull/2

I couldn't get the `FakeContract` fixture to work as I wasn't sure what to bind to the `MyContract` template argument.  

I tried importing ERC721.json from the "artifacts" folder but that yielded an inscrutable run-time error

I was still able to reduce this to a single line with this style: 
`const myFake = await smock.fake('MyContract');`

Reply:
ah seems you did this already.  nonetheless, i changed this line so that it returns `nftOwner.address` when `ownerOf` is called with arg: `tokenId`

change is here: 
https://github.com/johnmosesman/nftrusty/pull/2/commits/aab0468d45b3283004049b88115c1dc5ed2b6b75#diff-69ed30ac6a9db3ff7e25f33232aaeb3512192f8d6d84afe5702aa5dfd48f20beR25

Reply:
Yeah it helped cleanup some other stuff I was already doing, but I like the inclusion of the `with` (just merged that pr)